libs/parallelism/algorithms/include/hpx/parallel/algorithms/replace.hpp

```diff
@@ -346,124 +709,68 @@ namespace hpx { namespace parallel { inline namespace v1 {
-    ///                     the iterations.
-    /// \param first        Refers to the beginning of the sequence of elements
-    ///                     the algorithm will be applied to.
-    /// \param last         Refers to the end of the sequence of elements the
-    ///                     algorithm will be applied to.
-    /// \param dest         Refers to the beginning of the destination range.
-    /// \param old_value    Refers to the old value of the elements to replace.
-    /// \param new_value    Refers to the new value to use as the replacement.
-    /// \param proj         Specifies the function (or function object) which
-    ///                     will be invoked for each of the elements as a
-    ///                     projection operation before the actual predicate
-    ///                     \a is invoked.
-    ///
-    /// The assignments in the parallel \a replace_copy algorithm invoked
-    /// with an execution policy object of type \a sequenced_policy
-    /// execute in sequential order in the calling thread.
-    ///
-    /// The assignments in the parallel \a replace_copy algorithm invoked
-    /// with an execution policy object of type \a parallel_policy or
-    /// \a parallel_task_policy are permitted to execute in an unordered
-    /// fashion in unspecified threads, and indeterminately sequenced
-    /// within each thread.
-    ///
-    /// \returns  The \a replace_copy algorithm returns a
-    ///           \a hpx::future<tagged_pair<tag::in(FwdIter1), tag::out(FwdIter2)> >
-    ///           if the execution policy is of type
-    ///           \a sequenced_task_policy or
-    ///           \a parallel_task_policy and
-    ///           returns \a tagged_pair<tag::in(FwdIter1), tag::out(FwdIter2)>
-    ///           otherwise.
-    ///           The \a copy algorithm returns the pair of the input iterator
-    ///           \a last and the output iterator to the
-    ///           element in the destination range, one past the last element
-    ///           copied.
-    ///
-    template <typename ExPolicy, typename FwdIter1, typename FwdIter2,
-        typename T1, typename T2, typename Proj = util::projection_identity,
-        HPX_CONCEPT_REQUIRES_(hpx::is_execution_policy<
-            ExPolicy>::value&& hpx::traits::is_iterator<FwdIter1>::value&&
-                traits::is_projected<Proj, FwdIter1>::value&&
-                    traits::is_indirect_callable<ExPolicy, std::equal_to<T1>,
-                        traits::projected<Proj, FwdIter1>,
-                        traits::projected<Proj, T1 const*>>::value)>
-    typename util::detail::algorithm_result<ExPolicy,
-        hpx::util::tagged_pair<tag::in(FwdIter1), tag::out(FwdIter2)>>::type
-    replace_copy(ExPolicy&& policy, FwdIter1 first, FwdIter1 last,
-        FwdIter2 dest, T1 const& old_value, T2 const& new_value,
-        Proj&& proj = Proj())
+    // clang-format off
+    template <typename ExPolicy, typename FwdIter1, typename Sent,
```

Comment:
I know this PR has been merged but did you add typename Sent by mistake? Since last is of type FwdIter1.